doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -778,13 +788,13 @@
 
     | before     | after   |
     |:----------:|:-------:|
-    | `kOff`     | `Off`   |
-    | `kFatal`   | `Fatal` |
-    | `kError`   | `Error` |
-    | `kWarn`    | `Warn`  |
-    | `kInfo`    | `Info`  |
-    | `kDebug`   | `Debug` |
-    | `kVerbose` | `Trace` |
+    | `kOff`     | `OFF`   |
+    | `kFatal`   | `FATAL` |
+    | `kError`   | `ERROR` |
+    | `kWarn`    | `WARN`  |
+    | `kInfo`    | `INFO`  |
+    | `kDebug`   | `DEBUG` |
+    | `kVerbose` | `TRACE` |
```

Comment:
Please revert this. The old writing was correct.

Reply:
@elBoberido Good catch! Fixed in #2419